.github/workflows/test_local_upgrade.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Local Upgrade Tests"
+
+on:
+  pull_request:
+#  release:
+#    types:
+#      - prereleased
+#  workflow_dispatch:
```

Comment:
Triggers will be set properly before merge. Until then, I'd like to keep the PR trigger to ease the DX.